cpp/ps0004/problem solve 4.cpp

```diff
@@ -2,22 +2,27 @@
 using namespace std;
 int main()
 {
+    // prompting user for number 1.5/1.5
     float num;
     cout << "Enter a number from 32 to 126:\n";
     cin >> num;
```

Comment:
- [x] declared a variable to store the user input
- [x] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input